site/news/index.html

```diff
@@ -208,8 +208,9 @@ <h3 class="entry-title"><a href="/news/next-official-release-spark-3.1.1.html">N
       <div class="entry-date">January 7, 2021</div>
     </header>
     <div class="entry-content"><p>The next official Spark release is Spark 3.1.1 instead of Spark 3.1.0.
-There was an accident during Spark 3.1.0 RC1 preparation,
-see <a href="http://apache-spark-developers-list.1001551.n3.nabble.com/VOTE-Release-Spark-3-1-0-RC1-td30524.html">[VOTE] Release Spark 3.1.0 (RC1)</a> in the Spark dev mailing list.</p>
+There was a technical issue during Spark 3.1.0 RC1 preparation,
+see <a href="https://www.mail-archive.com/dev@spark.apache.org/msg27133.html">[VOTE] Release Spark 3.1.0 (RC1)</a> in the Spark dev mailing list.</p>
```

Comment:
Oh, yeah. I think I forgot to generate again after fixing MD file. Thanks for fixing this.